port_ocean/core/defaults/initialize.py

```diff
@@ -205,12 +248,9 @@ async def _initialize_defaults(
         raise ExceptionGroup(str(e), e.errors)
 
 
-def initialize_defaults(
+def initialize_integration(
     config_class: Type[PortAppConfig], integration_config: IntegrationConfiguration
 ) -> None:
-    try:
-        asyncio.new_event_loop().run_until_complete(
-            _initialize_defaults(config_class, integration_config)
-        )
-    except Exception as e:
-        logger.debug(f"Failed to initialize defaults, skipping... Error: {e}")
+    asyncio.new_event_loop().run_until_complete(
```

Comment:
we have a file in the cli folder that use the create defaults make sure you are not breaking itz

Reply:
yeah it works